### PR TITLE
polish the cast op doc

### PR DESCRIPTION
**Pull request by tensor-tang**
fix 5th of https://github.com/PaddlePaddle/FluidDoc/issues/672

### CODE REVIEW

File: paddle/fluid/operators/cast_op.cc [hunk 1]
```diff
@@ -30,7 +30,8 @@ class CastOpProtoMaker : public framework::OpProtoAndCheckerMaker {
 Cast Operator.
 
 This Operator casts the input tensor to another data type and
-returns tha Output Tensor.
+returns tha Output Tensor. It's not meaningful if the output dtype equals
```
Comment (shanyi15) [inline paddle/fluid/operators/cast_op.cc]: typo: tha
not meaningful=>meaningless

File: python/paddle/fluid/layers/tensor.py [hunk 2]
```diff
@@ -142,7 +142,8 @@ def create_global_var(shape,
 def cast(x, dtype):
     """
     This layer takes in the Variable :attr:`x` with :attr:`x.dtype` and casts
-    it to the output with :attr:`dtype`.
+    it to the output with :attr:`dtype`. It's not meaningful if the output
```
Comment (shanyi15) [inline python/paddle/fluid/layers/tensor.py]: 不确定:attr:`dtype`后面不空格直接跟句号会不会在官网显示乱码，如果可以的话，请用预览工具预览
note meaningful=>meaningless

File: paddle/fluid/operators/cast_op.cc (hunk 1 above)
Comment (tensor-tang) [reply paddle/fluid/operators/cast_op.cc]: Thx

File: python/paddle/fluid/layers/tensor.py (hunk 2 above)
Comment (tensor-tang) [reply python/paddle/fluid/layers/tensor.py]: 这个改之前就是这样，应该不会有问题吧

**Review by shanyi15 — APPROVED** [review]
lgtm comments

**Review by panyx0718 — APPROVED** [review]

**Pull request merged**